### DISCUSSION TITLE
Runtime directory does not exist

### DISCUSSION THREAD

**Discussion by Konglomneshued** [helix-editor/helix#7859] [Troubleshooting]
I have tried two installation methods, both of which result in this error:

```
Runtime directory does not exist: /home/untitled1/.config/helix/runtime
```
First installation method I used was from the AUR, the second was building from source as installing from AUR did not provide a working hx command.

I added this line to my .bashrc:
```
HELIX_RUNTIME=/home/untitled1/src/helix/runtime
```

And tried changing it to:
```
HELIX_RUNTIME=/home/untitled1/.config/helix/runtime
```

Upon checking, there is no src folder in my home.  There is also no runtime in ~/.config/helix/

Please, wtf.

**Comment by the-mikedavis** [comment]
That's a warning rather than an error: it's saying that one of the default runtime directory locations isn't present. The runtime files are present as long as one of the dictories in the "Runtime directories" list in `hx --health` is found and contains the files.

**Reply by Konglomneshued** [reply]
It lists them both, I just listed one of the errors.  I checked which hx and it says it's in .cargo/bin but there is nothing named runtime in there.

**Reply by the-mikedavis** [reply]
What's the output of `hx --health | head -n 15`?

**Reply by Konglomneshued** [reply]
```
Config file: default
Language file: default
Log file: /home/untitled1/.cache/helix/helix.log
Runtime directories: /home/untitled1/.config/helix/runtime;/home/untitled1/.cargo/bin/runtime
Runtime directory does not exist: /home/untitled1/.config/helix/runtime
Runtime directory does not exist: /home/untitled1/.cargo/bin/runtime
Clipboard provider: xsel
System clipboard provider: xsel

Language           LSP                DAP                Highlight          Textobject         Indent             
astro              None               None               ✘                  ✘                  ✘                  
awk                ✘ awk-language-…   None               ✘                  ✘                  ✘                  
bash               ✘ bash-language…   None               ✘                  ✘                  ✘                  
bass               ✘ bass             None               ✘                  ✘                  ✘                  
beancount          None               None               ✘                  ✘                  ✘    
```

**Reply by the-mikedavis** [reply]
If you are building from source you will need to either symlink the `runtime/` directory from the Helix repository or point the HELIX_RUNTIME environment variable at the `runtime/` directory from the Helix repo. So if you cloned Helix into `/home/untitled1/src/helix` you should be able to set `HELIX_RUNTIME=/home/untitled1/src/helix/runtime` or symlink that directory to `/home/untitled1/.config/helix/runtime`

**Reply by Konglomneshued** [reply]
I already did that, see the post.

**Reply by jonsaw** [reply]
It needs to be 

```
export HELIX_RUNTIME=/home/untitled1/src/helix/runtime
```

Did you prefix with an `export` in your `.bashrc`?

**Reply by atongrun** [reply]
I added the environment variable, but still: Runtime directory does not exist: /home/ss/.cargo/bin/runtime.
Any other solutions?
